src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,23 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
```

Comment:
I don't think you need the getUsers and method here

Reply:
made new commit